sql/remote_content_uptake_v1.sql

```diff
@@ -0,0 +1,74 @@
+CREATE TEMP FUNCTION
+  udf_get_key_from_map_entry(entry ANY TYPE,
+    key ANY TYPE) AS (
+  IF
+    (entry.key = key,
+      entry.value,
+      NULL));
+  --
+SELECT
+  submission_date_s3 AS submission_date,
+  uptake.key AS `source`,
+  -- These timestamps help us display a date range when we present the
+  -- data. It helps to "remind" you when something first started appearing.
+  MIN(`timestamp`/1000000000) AS min_timestamp,
+  MAX(`timestamp`/1000000000) AS max_timestamp,
+  SUM(udf_get_key_from_map_entry(status,
+      'up_to_date')) AS up_to_date,
```

Comment:
Can we add some commentary here that links to the code where these various keys are defined? It would be nice to easily make that jump in the future to confirm we aren't missing cases, etc.

Reply:
added comment: 
```
  -- Create a column for each label on UPTAKE_REMOTE_CONTENT_RESULT_1 in
  -- https://hg.mozilla.org/mozilla-central/file/tip/toolkit/components/telemetry/Histograms.json
```